src/sb_pointcloud_processing/CMakeLists.txt

```diff
@@ -81,6 +89,7 @@ target_link_libraries(line_extractor_node
     ${catkin_LIBRARIES}
     ${PCL_COMMON_LIBRARIES}
     ${PCL_IO_LIBRARIES}
+    gomp
```

Comment:
Just noticed this, do we still need to link against `gomp`?

Reply:
nope. removing it